backend/app.js

```diff
@@ -5,15 +5,40 @@ const port = 3001;
 const db = require("./AnnouncementsQueries");
 const cors = require("cors");
 const fetch = require("node-fetch");
-const { userId, basicAuth } = require("./config");
+const { userId, basicAuth, loginPassword, access_token } = require("./config");
```

Comment:
change access_token to accessToken, also post updated config file to slack channel